general-superstaq/general_superstaq/_models.py

```diff
@@ -0,0 +1,291 @@
+"""Data models used when communicating with the superstaq server."""
+
+# pragma: no cover
```

Comment:
This will need to be removed once client-side starts using these models.

Reply:
Remind me why these aren't server side for now? https://github.com/Infleqtion/client-superstaq/pull/1106/files doesn't use them either.

Reply:
https://github.com/Infleqtion/client-superstaq/pull/1106/files does use the models when handling server responses (it validates that the response is in the expected format). See for instance gss.client._SuperstaqClient_v0_3_0.create_job()`
 
 I'm adding them to client side now so that I can import them server side.

Reply:
Because for now there are no tests that use any of these models (because they are just data structures)